src/main/java/shop/service/impl/OrderServiceImpl.java

```diff
@@ -1,6 +1,52 @@
 package shop.service.impl;
 
+import shop.dao.OrderDAO;
+import shop.dao.impl.OrderDAOImpl;
+import shop.entity.Order;
 import shop.service.OrderService;
 
-public class OrderServiceImpl {
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class OrderServiceImpl implements OrderService {
+    private Map<String, Order> orderMap = new HashMap<>();
+    private OrderDAO orderDAO = new OrderDAOImpl();
+
+    @Override
+    public void addProductToOrder(String productId) {
+
+        orderMap.put(productId, (Order) orderDAO);
+    }
+
+    @Override
+    public void getOrder(String orderId) {
+
+    }
+
+    @Override
+    public List<Order> getAllOrdersForUser(String usersId) {
+        List<Order> allOrders = new ArrayList<>();
+        if (usersId.equals(orderDAO)) {
+            allOrders = orderMap.values().stream()
```

Comment:
change to 
List<Order> orders = orderDaoImpl.getOrdersByUserId();

return  orders.stream()
                    .filter(order -> order.getUserId().equals(orderDAO))
                    .collect(Collectors.toList());